README.md

```diff
@@ -8,29 +8,31 @@ This library builds on top of the [Microsoft.OpenAPI.NET](https://github.com/mic
 
 ## Getting started
 
+### Generating SDKs
+
+1. Install the required tools for your language. (refer to the [Supported Languages](#supported-languages) table under the **Required tools & dependencies** column)
```

Comment:
This one makes me feel uneasy, but it just might be me, being a silly sausage.

- I want to autogenerate a "Client" for some OpenAPI endpoint(s)
- "Client" = "a lightweight set of strongly typed classes that layer over a set of core HTTP libraries and produce an intuitive and discoverable way of creating HTTP requests."
- To do this, I need some 'tool' that can generate the "Client"
- So before I can use the tool, I need to install stuff, first?

I feel like this line is a barrier to entry. And confuses people.

Personally, I don't want to install -anything- extra .. just the single tool to generate my "Client".

I think this step is actually a _sub-step_ based upon the tool you wish to install.

For example:
- Use the Docker image to generate Clients: only tool here is to have docker desktop installed. Pretty much, implied.
- Use the binaries: I have no idea how the binaries are generated. Do I need .NET installed then? If yes, this is where this should be mentioned. Are the binaries "self contained"? then I don't need anything to install, right? Path of least resistance, etc.
- Dotnet tool: I'll definitely need .NET < what version? > installed. That's ok, because only -now- have I decided on my tool and this is also based upon the fact that I'm happy with this dependency.
- Cone repo: I'll need git and .NET etc. For the hardcore.

So, TL;DR;
- nuke this line
- put this info into the DOWNLOADS page where people can determine which tool they wish to use.

Reply:
yeah you're right, this is super redundant with the next line, I think it's a consequence of multiple updates.
```suggestion
```